Listener/CBLHTTPConnection.m

```diff
@@ -110,6 +112,26 @@ - (NSArray *)sslIdentityAndCertificates {
 }
 
 
+- (NSURL*) remoteURL {
+    if (!_builtRemoteURL) {
+        NSString* addr = asyncSocket.connectedHost;
+        if (![addr isEqualToString: @"127.0.0.1"]  && ![addr isEqualToString: @"::1"]) {
```

Comment:
`source` is supposed to be set when a revision comes from an external source, generally a replication. If it's coming from localhost, it's presumably the same app, via PhoneGap or something like that.